### PR TITLE
Nits

### PR DESCRIPTION
**Pull request by JSwambo**
Some language and link fixes while reviewing practical-revault in its entirety.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,13 @@
 # Practical revault
 
-A protocol sketch for communication between the different entities in the [Revault
-architecture](revault.pdf).
+A specification for the initialization and operation of the [Revault](revault.pdf) custody protocol.
```
Comment (darosior) [inline README.md:3]: Not yet for the initialization tho :(

File: README.md [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
-Anyone is welcome for bikeshedding with us, to raise issues for either asking questions
-about, or pointing out actual issues with the "protocol".
-
-
-JSON is used to straightforwardly describe messages here but the actual encoding
-isn't decided yet.
+moving forward with the implementation.
 
+Anyone is welcome to contribute by raising issues for either asking questions
+about, or pointing out actual issues with the protocol.
```
Comment (darosior) [inline README.md:10]: Nice cleanup

File: messages.md [hunk 3]
```diff
@@ -461,7 +461,7 @@ We use a timestamp as watchtowers might accept the same spending attempt in the
 #### `get_spend_opinions`
 
 Sent by a manager when polling for watchtowers agreement regarding the spend
-attempt identified by `vault_id`.
+attempt identified by `spend_transaction_id`.
```
Comment (darosior) [inline messages.md:464]: The field is named `id`

File: messages.md [hunk 4]
```diff
@@ -288,7 +288,7 @@ until the Bitcoin network deploys [package relay][package_relay].
 #### `sig`
 
 Sent by a stakeholder wallet at any point in time to share the signature for a transaction
-with all participants.
+with all other steakholders and all managers (in the case of unvault and cancel transactions).
```
Comment (darosior) [inline messages.md:291]: Lol i'm sure you did that on purpose

Comment (darosior) [reply messages.md:291]: You troll

File: messages.md [hunk 5]
```diff
@@ -529,7 +529,7 @@ transaction to the watchtowers.
 #### `get_spend_validations`
 
 Sent by a manager when polling for watchtowers acknowledgement of the fully signed
-transaction spending the vault identified by `vault_id`.
+transaction spending the vault identified by `spend_transaction_id`.
```
Comment (darosior) [inline messages.md:532]: Same, `id`. But nice catch of this leftovers

File: transactions.md [hunk 6]
```diff
@@ -20,7 +21,7 @@ and `X` the CSV value in the unvault transaction.
 
 ## vault_tx
 
-The deposit transaction.
+The transaction which deposits into the vault.
```
Comment (darosior) [inline transactions.md:24]: Why ?

File: transactions.md [hunk 7]
```diff
@@ -7,7 +7,8 @@ describe outputs spending policies.
 
 We denote `N` the number of stakeholders, `M` the number of fund managers (allowed
 to unlock the unvault transaction output along with the cosigning servers after a timeout),
-and `X` the CSV value in the unvault transaction.
+and `X` the CSV value in the unvault transaction. Note that a sequence number of `0xfffffffd` 
+signals replace-by-fee (RBF).
```
Comment (darosior) [inline transactions.md:11]: If you want to explicit that transactions are signaling RBF, i'm fine with it. But i'm not with stating the obvious in the introduction.

Maybe "All transactions that we create signal for RBF" ?

File: transactions.md (hunk 6 above)
Comment (darosior) [reply transactions.md:24]: Hmm actually we should rename it to deposit anyways so that's fine

File: transactions.md [hunk 8]
```diff
@@ -103,8 +104,8 @@ path, only spendable after `X` blocks.
 
 ## cancel_tx
 
-This transaction spends the `unvault_tx` using the N-of-N path and pays back to a
-vault txout (it is therefore another deposit transaction).
+The transaction which spends the [`unvault_tx`](unvault_tx) `output[0]` using the N-of-N path and 
+pays back to a vault txout (it is therefore another vault deposit transaction).
```
Comment (darosior) [inline transactions.md:108]: Nice

File: transactions.md [hunk 9]
```diff
@@ -129,15 +130,15 @@ vault txout (it is therefore another deposit transaction).
...
 ## emergency_txs
 
-Emergency transactions are used as deterrents against threat. They lock coins to what we
-call an EDV (Emergency Deep Vault): a script chosen by the participants and kept
-obfuscated by the properties of P2WSH, as the emergency transactions are never meant
-to be used.
+Emergency transactions are used as deterrents against threats targetting stakeholders' 
+funds. They lock coins to what we call an EDV (Emergency Deep Vault): a script chosen 
+by the participants and kept obfuscated by the properties of P2WSH, as the emergency 
+transactions are never meant to be used.
```
Comment (darosior) [inline transactions.md:136]: I agree that my "against threat" was not specific enough. However "threats targetting stakeholders' funds" is no better ? All threats are against the stakeholders' funds.

File: transactions.md [hunk 10]
```diff
@@ -129,15 +130,15 @@ vault txout (it is therefore another deposit transaction).
...
 
 
 ### vault_emergency_tx
 
-This transaction spends the `vault_tx` to the EDV by the `N`-of-`N` path.
+The transaction which spends the [`vault_tx`](vault_tx) output to the EDV by the `N`-of-`N` path.
```
Comment (darosior) [inline transactions.md:141]: :+1:

**Review by darosior — COMMENTED** [review]
Globally really nice, thanks!
Some small disagreements on some parts though

File: README.md (hunk 1 above)
Comment (darosior) [reply README.md:3]: But don't change it it's fine

File: transactions.md (hunk 7 above)
Comment (JSwambo) [reply transactions.md:11]: What is obvious to you is not obvious to others? I had to remind myself about how this is done while reading..

File: transactions.md (hunk 9 above)
Comment (JSwambo) [reply transactions.md:136]: Not really. Threats can be against stakeholders themselves, privacy, operational continuity...

Comment (darosior) [reply transactions.md:136]: Right

File: transactions.md (hunk 7 above)
Comment (darosior) [reply transactions.md:11]: It is to the expected reader of the document. Expliciting it seems to me as explaining the inner working of an engine at the top of an engine specification document, it's not bad as such: just inappropriate

File: transactions.md [hunk 11]
```diff
@@ -18,9 +19,9 @@ and `X` the CSV value in the unvault transaction.
     - [unvault_emergency_tx](#unvault_emergency_tx)
 
 
-## vault_tx
+## deposit_tx
```
Comment (darosior) [inline transactions.md:22]: Thanks, i was going to make a PR for this.

**Review by darosior — APPROVED** [review]
ACK 397641855c48cea232bae7c909e7c115d42a3384

**Comment by darosior** [comment]
Next time, please just use [more meaningful commit names](https://github.com/re-vault/revaultd/blob/master/CONTRIBUTING.md#workflow) than 'Update README.md" for example.

**Pull request merged**